start.R

```diff
@@ -243,6 +243,10 @@ if (any(c("--reprepare", "--restart") %in% flags)) {
     slurmConfig <- "direct"
     message("\nTrying to compile ", nrow(scenarios), " selected runs...")
     lockID <- gms::model_lock()
+    if (length(missingInputData()) > 0) {
+      # try to fix missing input data, but only once at the beginning, not for every scenario
+      updateInputData(readDefaultConfig("."), remindPath = ".", gamsCompile = FALSE)
```

Comment:
Is it a problem, that at this point the cfg is not yet configured and we have to use the defaults?

Reply:
No. Because if one of the scenarios has a different `cfg`, [this line](https://github.com/remindmodel/remind/blob/develop/scripts/start/updateInputData.R#L24) will trigger a reload anyway. So what might happen if you want to compile a REMIND-EU config file _and_ some input files are missing, that it first downloads REMIND-H12 and then again REMIND-EU. But well, it doesn't break anything.